frontend/src/components/Auth/Login.tsx

```diff
@@ -0,0 +1,27 @@
+import { User } from "@/types/User";
+import { FC } from "react";
+import { useForm } from "react-hook-form";
+import Form from "../Form";
+import RHFInput from "../Form/RHFinput";
+import useLogin from "@/hooks/useLogin";
+
+type loginProps = {
+  setShowAuth: (showAuth: boolean) => void;
+};
+
+const Login: FC<loginProps> = ({ setShowAuth }) => {
+  const methods = useForm();
+  const { mutateAsync } = useLogin();
```

Comment:
It's better to rename `mutateAsync` function like this `const { mutateAsync: login } = useLogin();`. This will make the code more redable. Same goes for `Register` component.